common/api-review/messaging-exp.api.md

```diff
@@ -25,7 +36,31 @@ export function getToken(messaging: FirebaseMessaging, options?: {
     swReg?: ServiceWorkerRegistration;
 }): Promise<string>;
 
-export { MessagePayload }
+// @public (undocumented)

```

Comment:
Should these have documentation? It looks like they didn't previously.

Reply:
They should. @zwu52, can you please add missing comments for the public APIs. Let's do it in a separate PR though.